maturity-level-two/tests/Codit.UnitTest/Controllers/carControllerTest.cs

```diff
@@ -27,7 +27,7 @@ public CarControllerTest()
         public async Task GetCars_test()
         {
             //act
-            var cars = (await _controller.GetCars()) as OkObjectResult;
+            var cars = (await _controller.GetCars(null)) as OkObjectResult;
```

Comment:
Why an explicit null argument ?

Reply:
in the customization controller:

```C#
public async Task<IActionResult> GetCars([FromQuery] CarBodyType? bodyType)
```
The GetCars operation supports an optional query for the car body type. If provided, the method returns only the cars of the given body type. If not, then the parameter from the query will be null. Therefore the GetCars method has a nullable parameter (CarBodyType?) which must be provided. In the test case, I don't use the CarBodyType query so the parameter should be explicitely null, otherwise I get an error saying that an argument must be provided.